scipy/stats/stats.py

```diff
@@ -5194,7 +5194,7 @@ def kruskal(*args, **kwargs):
     for i in range(num_groups):
         ssbn += _square_of_sums(ranked[j[i]:j[i+1]]) / float(n[i])
 
-    totaln = np.sum(n)
+    totaln = np.sum(n, dtype='uint64')
```

Comment:
Minor: Can in principle just use `dtype=float` straight away. Otherwise it gets converted in the next line (int + uint -> float, an obscure numpy feature).

Reply:
Changed.